Jenkinsfile_CNP

```diff
@@ -122,6 +122,12 @@ withPipeline(type, product, component) {
     """
     }
 
+    afterSuccess('akschartsinstall') {
+        onPR {
+            githubUpdateDeploymentStatus(githubCreateDeployment(), "https://ia-case-api-pr-${CHANGE_ID}-aip-frontend.preview.platform.hmcts.net")
```

Comment:
This interferes with the "normal" preview link that is created. Change it as follows and keep an eye on the deployment buttons.

```suggestion
            githubUpdateDeploymentStatus(githubCreateDeployment("-AIP"), "https://ia-case-api-pr-${CHANGE_ID}-aip-frontend.preview.platform.hmcts.net")
```